README.md

```diff
@@ -9,23 +9,22 @@
 [![npm](https://img.shields.io/npm/dm/levelup.svg)](https://www.npmjs.com/package/levelup)
 
   * <a href="#intro">Introduction</a>
-  * <a href="#platforms">Tested &amp; supported platforms</a>
-  * <a href="#basic">Basic usage</a>
+  * <a href="#platforms">Supported Platforms</a>
+  * <a href="#usage">Usage</a>
   * <a href="#api">API</a>
   * <a href="#promises">Promise Support</a>
-  * <a href="#import">Import with Type Definitions</a>
+  * <a href="#import">ES6 Import</a>
   * <a href="#events">Events</a>
-  * <a href="#extending">Extending levelup</a>
-  * <a href="#multiproc">Multi-process access</a>
-  * <a href="#support">Getting support</a>
+  * <a href="#extending">Extending `levelup`</a>
+  * <a href="#multiproc">Multi-process Access</a>
+  * <a href="#support">Getting Support</a>
   * <a href="#contributing">Contributing</a>
-  * <a href="#license">Licence &amp; copyright</a>
+  * <a href="#license">Licence and Copyright</a>
```

Comment:
Just "License"

Reply:
Aye, just noticed it was misspelled as well.

Reply:
Both are correct, license is just more common in OSS